saleor/graphql/checkout/mutations/checkout_lines_update.py

```diff
@@ -113,6 +127,26 @@ def clean_input(
                     }
                 )
 
+        # # if same variant occur in multiple lines `lineId` parameter have to be used
+        variants_ids_in_existing_lines = [line.variant.pk for line in lines]
+        for line in checkout_lines_data:
+            if (
+                line.variant_id
+                and variants_ids_in_existing_lines.count(line.variant_id) > 1
+            ):
+                message = (
+                    f"Variant #{line.variant_id} occur in multiple lines. "
```

Comment:
Just to confirm. Is `variant_id` graphql ID?

Reply:
I had to move this error to another place, but now I used `variants` parameter as @fowczarek suggested and global IDs are returned.